### PR TITLE
[M10][M11][M12] Fix all texts

### PR DESCRIPTION
**Pull request by Susucre**

### CODE REVIEW

File: Mage.Sets/src/mage/cards/t/Triskelion.java
Comment (JayDi85) [inline Mage.Sets/src/mage/cards/t/Triskelion.java:40]: Can use auto-generation here. See additional params of the effect. `setText` method should be used as little as possible (someday there are will be verify tests to collect stats of `setText` usages).

shot_230902_052549

Comment (Susucre) [reply Mage.Sets/src/mage/cards/t/Triskelion.java:40]: Oh missed that.
How about having a `setReplaceThisInRule(String replacement)` for stuff like that? That way we set {this} in the rule, and if set at the end of getRules we replace any {this} by the replacement text? On a per-effect basis, no need to have all effect have that method.
I feel like we don't really want rule logic in the effect constructor (they do not modify anything), and could use it there for the "sacrifice it" from the Phantasmal creatures.

Comment (Susucre) [reply Mage.Sets/src/mage/cards/t/Triskelion.java]: Ok I've pushed my proposal on that for the `SacrificeSourceEffect` text generation.
@xenohedron mentioned on Discord they have opinions on the matter but no time right now to express them, so let's wait on discussion to settle before merging this one.

File: Mage/src/main/java/mage/abilities/effects/common/SacrificeSourceEffect.java
Comment (xenohedron) [inline Mage/src/main/java/mage/abilities/effects/common/SacrificeSourceEffect.java:61]: I don't like overriding `getText(Mode mode)` when the mode parameter isn't needed. Instead, add back the `staticText` in the constructor and set it direclty in the `setSourceName` method as well, with no need for storing the sourceName in a field.

File: Mage/src/main/java/mage/abilities/effects/common/ReturnFromExileForSourceEffect.java
Comment (xenohedron) [inline Mage/src/main/java/mage/abilities/effects/common/ReturnFromExileForSourceEffect.java:93]: I know false is the default, but generally the style I've seen is to take a boolean parameter

File: Mage/src/main/java/mage/abilities/effects/common/ReturnFromExileForSourceEffect.java
Comment (xenohedron) [inline Mage/src/main/java/mage/abilities/effects/common/ReturnFromExileForSourceEffect.java:12]: Let's not use this exception. IllegalArgumentException maybe?

File: Mage.Verify/src/test/java/mage/verify/VerifyCardDataTest.java
Comment (xenohedron) [inline Mage.Verify/src/test/java/mage/verify/VerifyCardDataTest.java:2381]: I'm not a big fan of this. I think I'd rather leave the text as it is for now, and separately consider if it's possible to handle the abilities as effects that can be combined rather than the current implementation that forces them to be separate abilities.

File: Mage.Verify/src/test/java/mage/verify/VerifyCardDataTest.java
Comment (xenohedron) [inline Mage.Verify/src/test/java/mage/verify/VerifyCardDataTest.java:66]: The reason to not use * here in master is it'll make the logs unwieldy. Someday, once we address a bunch more text discrepancies.

File: Mage.Sets/src/mage/cards/p/PhantasmalBear.java
Comment (xenohedron) [inline Mage.Sets/src/mage/cards/p/PhantasmalBear.java:29]: I was trying to handle these automatically in `TriggeredAbilityImpl` but these ones are getting missed right now as the trigger phrase contains "of a". I have some ideas for how to improve the automatic detection. Goal is to avoid having this sort of manual override on the effect.

**Review by xenohedron — CHANGES_REQUESTED** [review]
let me know what you think of my comments. The changes to Seismic Strike, Thunder Strike, and Triskelion are fine but I think there are might be better ways to deal with the others, so I'd rather not merge this as is.

**Pull request closed without merging**
